tradesman/model_creation/import_political_subdivisions.py

```diff
@@ -0,0 +1,234 @@
+import sqlite3
+from tempfile import gettempdir
+import warnings
+from fiona import listlayers
+import geopandas as gpd
+import pandas as pd
+from shapely.ops import unary_union
+import requests
+from aequilibrae.project import Project
+from os.path import join, isfile
+from shapely.geometry import Polygon, MultiPolygon
+import pycountry
+import urllib
+from numpy import arange
+
+
+class ImportPoliticalSubdivisions:
```

Comment:
I like the class, just not the fact that you are immediately telling it to do things. I would separate in two